src/types/User.ts

```diff
@@ -1,6 +1,36 @@
+// export interface User {
```

Comment:
Consider removing the commented-out code at the beginning of the file, as it is not needed and can make the code harder to read 🧹